shared/src/ledger/native_vp.rs

```diff
@@ -61,9 +73,9 @@ where
     H: StorageHasher,
 {
     /// Storage prefix iterators.
-    pub iterators: PrefixIterators<'a, DB>,
+    pub iterators: Rc<RefCell<PrefixIterators<'a, DB>>>,
```

Comment:
I don't think `Rc` is needed here and for the `gas_meter` as each native VP instance should have their own instances